src/icons/ArrowUp.tsx

```diff
@@ -0,0 +1,36 @@
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
+
+const pathClassName = classnames(
+  stroke('stroke-primary-light'),
+  strokeWidth('stroke-4')
+)
+
+export default function ({ small }: { small?: boolean }) {
```

Comment:
Let's call this `simpleArrow`, like we did on SC

Reply:
Done